src/net/java/sip/communicator/impl/configuration/ConfigurationActivator.java

```diff
@@ -89,19 +203,16 @@ public void start(BundleContext bundleContext)
 
     private boolean usePropFileConfigService(BundleContext bundleContext)
     {
-        if (Boolean.getBoolean(PNAME_USE_PROPFILE_CONFIG))
+        if (Boolean.valueOf(defaultProperties.get(PNAME_USE_PROPFILE_CONFIG)))
```

Comment:
No. This is intentionally a system property and not a Jitsi configuration property. It is only intended to be used by Jitsi Meet components (e.g. Jigasi) embedding some parts of Jitsi Desktop.